test/lib/y2network/proposal_settings_test.rb

```diff
@@ -100,34 +106,61 @@ def stub_features(features)
         end
 
         context "and the machine is a laptop" do
-          it "sets :network_manager as the backend to be used" do
-            expect(created_instance.backend).to eql(:network_manager)
+          it "returns :network_manager as the default backend" do
+            expect(subject.default_backend).to eql(:network_manager)
           end
         end
 
         context "and the machine is not a laptop" do
           let(:is_laptop) { false }
-          it "sets :wicked as the backend to be used" do
-            expect(created_instance.backend).to eql(:wicked)
+          it "returns :wicked as the default backend" do
+            expect(subject.default_backend).to eql(:wicked)
           end
         end
       end
 
       it "initializes the default network backend from the product control file" do
-        expect(described_class.create_instance.backend).to eql(:network_manager)
+        expect(subject.default_backend).to eql(:network_manager)
         stub_features("network" => { "network_manager" => "" })
-        expect(described_class.create_instance.backend).to eql(:wicked)
+        expect(subject.default_backend).to eql(:wicked)
       end
     end
 
     it "logs which backend has been selected as the default" do
       allow_any_instance_of(described_class).to receive(:log).and_return(logger)
-      expect(logger).to receive(:info).with(/backend is: network_manager/)
-      described_class.create_instance
+      expect(logger).to receive(:info).with(/backend is: wicked/)
+      subject.default_backend
+    end
+  end
+
+  describe "#current_backend" do
+    let(:selected_backend) { :wicked }
+    let(:default_backend) { "wicked_or_nm" }
```

Comment:
This seems to be a leftover, isn't it?

Reply:
Not really, but it should be used in the line below :)